### DISCUSSION TITLE
how to start adb in pair as su and run script

### DISCUSSION THREAD

**Discussion by xxchizaxx** [termux/termux-app#3292] [Q&A] [answered]
I've been trying to start debugging wifi pairing for a while as root user to start a script to activate an app

I tried to run this line but it tells me that I don't have access sh /sdcard/Android/data/com.panda.scriptpatcher/files/activate.sh

**Comment by 2096779623** [comment, marked as answer]
```bash
pkg i tsu -y
tsu
adb pair ip:port
adb connect ip:port
adb shell #your command
```

**Reply by xxchizaxx** [reply]
thanks my friend
